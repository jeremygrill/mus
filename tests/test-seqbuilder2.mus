Int main(){
	
	Int i;
	Note n;
	Chord c;
	Seq s;

	for(i = 0; i < 20; i = i + 1){
		n = (5 * i, 10|10);
		c =  [n];
		s = s + c;
	}
	

	prints(s);

	return 0;
}
