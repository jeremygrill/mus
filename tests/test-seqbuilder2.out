$([(0,10|10)][(5,10|10)][(10,10|10)][(15,10|10)][(20,10|10)][(25,10|10)][(30,10|10)][(35,10|10)][(40,10|10)][(45,10|10)][(50,10|10)][(55,10|10)][(60,10|10)][(65,10|10)][(70,10|10)][(75,10|10)][(80,10|10)][(85,10|10)][(90,10|10)][(95,10|10)])
